src/model/version_list.rs

```diff
@@ -5,6 +5,7 @@ impl VersionList {
     pub fn latest_version(versions: Vec<Version>) -> Option<Version> {
         let mut sorted_versions = versions.to_vec();
         sorted_versions.sort_by(|a, b| a.uploaded_at.cmp(&b.uploaded_at));
+        sorted_versions.reverse();
```

Comment:
What about making the change in the line above, directly sorting it in the right order and saving the reversing operation?

```rust
sorted_versions.sort_by(|a, b| b.uploaded_at.cmp(&a.uploaded_at));
```

Reply:
yes, I was just thinking about that.